src/main/propertiesService.ts

```diff
@@ -112,3 +113,23 @@ export function deleteAllDocumentProperties(): void {
         throw new PropertiesServiceDeleteError(`Failed to delete all document properties: ${error.message}`)
     }
 }
+
+/**
+ * Logs all document properties of the current Google Sheets document.
+ *
+ * Retrieves and logs key-value pairs from the document properties using the 
+ * Apps Script Logger. Useful for debugging and auditing document metadata.
+ *
+ * @function logDocumentProperties
+ * @returns {void}
+ *
+ * @example
+ * logDocumentProperties();
+ */
+export function logDocumentProperties() {
+    var allProperties = documentProperties.getProperties();
+    for (var key in allProperties) {
+      Logger.log(key + ': ' + allProperties[key]);
```

Comment:
Nit: I know this is just a function for debugging but this code is missing type definitions

Reply:
Updated logDocumentProperties() with type definitions